var/spack/repos/builtin/packages/h5z-zfp/package.py

```diff
@@ -33,13 +33,31 @@ class H5zZfp(MakefilePackage):
     patch('Makefile.0.7.0.patch', when='@0.7.0')
```

Comment:
The `Makefile` patch is no longer needed now that the package.py is building with cmake